source/github-helpers/add-button-next-to-branch-select-menu.ts

```diff
@@ -0,0 +1,16 @@
+import select from 'select-dom';
+import elementReady from 'element-ready';
+
+export default async function addButtonNextToBranchSelectMenu(button: Element): Promise<void> {
```

Comment:
Suggestions for a shorter name welcomed :smile:

Reply:
appendToBranchMenu ?

Reply:
addAfterBranchSelector.